.github/workflows/ci.yml

```diff
@@ -131,7 +131,7 @@ jobs:
         run: |
           set PKG_CONFIG_PATH=%cd%\BUILD_win-amd64\INSTALL\lib\pkgconfig
           dir %PKG_CONFIG_PATH%
-          meson.exe setup . build -Dwith_xapian=false
+          meson.exe setup . build -Dwith_xapian=false -Dwerror=false
```

Comment:
Regarding the commit message - it contains a missing reference [3]. I guess it has to be [2].

Reply:
[3] is a link to a specific comment in [2].

Commit message has been update.